samples/system-test/v1p1beta1/notifications.test.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {SecurityCenterClient} =
+    require('@google-cloud/security-center').v1p1beta1;
+const {assert} = require('chai');
+const {describe, it, before} = require('mocha');
+const {execSync} = require('child_process');
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+
+const organizationId = process.env['GCLOUD_ORGANIZATION'];
+const orgName = 'organizations/' + organizationId;
+const pubsubTopic = 'projects/project-a-id/topics/notification-sample-topic';
+
+describe('Client with Notifications', async () => {
+  const createConfig = 'notif-config-test-node-create';
```

Comment:
This looks like it would be a risk for collisions.  What would happen if we ran the tests side by side?  One option here could be to use the `uuid` npm module to ensure unique names across test runs.

Reply:
Done.